core/src/themes/ionic.theme.default.md.scss

```diff
@@ -36,3 +36,9 @@ $item-md-color:                             var(--ion-item-color, $text-color) !
 // --------------------------------------------------
 $card-md-background:                        var(--ion-card-background, $item-md-background) !default;
 $card-md-color:                             var(--ion-card-color, var(--ion-item-color, $text-color-step-450)) !default;
+
+// Material Design Form Controls
+// --------------------------------------------------
+/// This value comes from the Material Design spec:
+/// https://m2.material.io/components/text-fields#outlined-text-field
+$form-control-md-disabled-opacity:          0.4;
```

Comment:
This value can be up for debate - the [Material Design 1 documentation](https://m1.material.io/components/text-fields.html#text-fields-states) (under Text field states -> Disabled) specifies it should be `.38`. The [Material Design 3 documentation](https://m3.material.io/components/text-fields/specs#983548e8-d974-42c5-ad1e-22c40cb08360) (next to Disabled at the bottom of the table) also uses `.38`. 

The [Material Design 2 documentation](https://m2.material.io/components/text-fields) does not explicitly list a percentage for the disabled text. I got the `0.4` value from [this image](https://github.com/ionic-team/ionic-framework/assets/6577830/741e1af8-d6fe-4bbe-a641-70a6e147ed78) under 'Outlined text field'. 

I am fine with using `.38` if we prefer that. I have updated this PR so it only needs to change in one place.

Reply:
If `0.38` passes contrast then I vote for using it. It would be great to use the latest version as the source of truth.

Reply:
I went and looked back at  the [Material 2 Design Kit in Figma](https://www.figma.com/community/file/778763161265841481) and the disabled selection control for checkbox & radio actually use `0.38` so I think it's safe to assume that is what the text fields would also use since Material Design 1 & 3 use that. From the Design Kit:

<img width="693" alt="Screenshot 2023-05-15 at 5 01 16 PM" src="https://github.com/ionic-team/ionic-framework/assets/6577830/a82643c8-40cf-4b85-adb9-b345536236e4">